projects/plots/plots/vizapp/pages/summary/sv.py

```diff
@@ -0,0 +1,216 @@
+import logging
+from pathlib import Path
+from typing import List, Optional
+
+import h5py
+import numpy as np
+from astropy.cosmology import Planck15 as cosmology
+from bilby.core.prior import PriorDict
+from bokeh.io import save
+from bokeh.layouts import gridplot
+from ledger.events import EventSet, RecoveredInjectionSet
+from ledger.injections import InjectionParameterSet
+from priors.priors import log_normal_masses
+
+from . import compute, utils
+from .gwtc3 import catalog_results
+
+SECONDS_PER_MONTH = 3600 * 24 * 30
+SECONDS_PER_YEAR = 60 * 60 * 24 * 365.25
+
+
+def get_prob(prior, ledger):
+    sample = dict(mass_1=ledger.mass_1_source, mass_2=ledger.mass_2_source)
+    return prior.prob(sample, axis=0)
+
+
+class SensitiveVolumePlot:
+    def __init__(
+        self,
+        background: EventSet,
+        foreground: RecoveredInjectionSet,
+        rejected_params: InjectionParameterSet,
+        mass_combos: List[tuple],
+        source_prior: PriorDict,
+        dt: Optional[float] = None,
+        max_far: float = 100 / SECONDS_PER_MONTH,
```

Comment:
It needs to be `SECONDS_PER_YEAR` here as well

Reply:
Does it? Im just defining the max_far as 100 per month

Reply:
Yeah, "need" was the wrong word. I was thinking that 1200 per year seemed like kind of a high FAR to go to. But that's a minor point

Reply:
got it, yeah thats probably right